docs/howto/openstack/create-new-account/cleura-cloud.md

```diff
@@ -0,0 +1,17 @@
+# Create a new account in Cleura Cloud
+
+To gain access to the {{extra.brand}} Cloud Control Panel, you first have to create a new account. For that, navigate to [https://cleura.cloud](https://cleura.cloud/login).
+
+At the bottom right-hand side of the page, click on the _Create account_ button. First, select the new account type (that would be _Company_ or _Private_), carefully type in a valid email address, and choose your country.
+
+At your leisure, please read the [City Network General Terms And Conditions](https://citycontrolpanel.com/files/City_Network_General_Terms_And_Conditions_City_Cloud.pdf) and our [Data Processing Agreement](https://citycontrolpanel.com/files/City_Network_CDPA.pdf). Agree to these documents (select _Yes_), check the _I'm not a robot_ box, and then click on the _Create_ button.
+
+You will immediately be redirected to the {{extra.brand}} Cloud Control Panel, and since you are logging in from a new account for the first time, you now have to take three simple steps.
+
+* __Step 1 - Confirm your email__. Check your inbox or your SPAM/junk folder for an email from `no-reply@cleura.com` with the subject _Thank you for your registration - Cleura Cloud_. Open that email and click on the link in the message body.
+
+* __Step 2 - Account information__. After clicking on the confirmation link you move on to step two, where you enter all relevant information that uniquely identifies the brand-new account. Type in, for example, a username for the account user, and make sure you define a strong password for them. (A password manager may come in handy.) Please note that all fields are mandatory, so take a little time and fill them in accordingly. Should you have a rebate code, do not forget to click on _I have a rebate code_ and type it in below. When you are done, click on the _Save_ button.
+
+* __Step 3 - Account verification__. While the new account is being created, and before it becomes fully operational, you have to take one last step toward verification. You do that either by entering valid credit card information or by placing a simple phone call. Should you choose to verify by credit card, rest assured that no charge will take place -- no money will be drawn from the card, in other words. On the other hand, if you prefer to verify by phone, you may certainly do so during business hours (08:00-17:00 CET). If you choose to call, please remember that you will be asked for the username of the new account, so have that piece of info handy.
```

Comment:
I know I am being an extreme stickler here, but the punctuation before "no money" needs to be an em dash (`---`), not an en dash (`--`). :)